drivers/serial/uart_stm32.c

```diff
@@ -1722,6 +1734,8 @@ static const struct uart_stm32_config uart_stm32_cfg_##index = {	\
 	.pcfg = PINCTRL_DT_INST_DEV_CONFIG_GET(index),			\
 	.single_wire = DT_INST_PROP_OR(index, single_wire, false), \
 	.tx_rx_swap = DT_INST_PROP_OR(index, tx_rx_swap, false),	\
+	.rx_invert = DT_INST_PROP_OR(index, rx_invert, false), \
+	.tx_invert = DT_INST_PROP_OR(index, tx_invert, false), \
```

Comment:
Since props are defined as boolean, you should be able to do:

```suggestion
	.rx_invert = DT_INST_PROP(index, rx_invert), \
	.tx_invert = DT_INST_PROP(index, tx_invert), \
```

Reply:
The problem here is again the _lpuart_, using the same driver, but it does not have the property.

Reply:
Well, it does seem supported on some series: G0/g4/h7/l0/l4/l5/u5/wb/wl.
Would you mind having a look on adding it as well for LPUART?
Note: you should be able to use exact same LL API (LL_USART_...)

Reply:
sure, will do